authors/apps/comments/utils/model_helpers.py

```diff
@@ -0,0 +1,55 @@
+import timeago
+from django.utils import timezone
+
+
+def get_single_comment_using_id(obj, id):
+    """
+    Method is a helper that retrieves an object or model
+    record basing on the model passed and the id provided
+    """
+    try:
+        comment = obj.objects.get(pk=id)
+        if comment:
```

Comment:
Thanks for the observation @missvicki but I choose to exclude and return a none because sometimes it returns me an error and the end result should be a none that's why I kinda of time cast it to a none.

Reply:
@FahdJamy I think I agree with @missvicki on this

Reply:
Thanks, @oma0256. Looking into that now.